src/couch/src/couch_httpd.erl

```diff
@@ -622,14 +622,16 @@ error_headers(#httpd{mochi_req=MochiReq}=Req, Code, ErrorStr, ReasonStr) ->
                 % redirect to the session page.
                 case ErrorStr of
                 <<"unauthorized">> ->
-                    case config:get("couch_httpd_auth", "authentication_redirect", undefined) of
+                    case chttpd_util:get_chttpd_auth_config(
+                        "authentication_redirect", "/_utils/session.html") of
```

Comment:
please use undefined as default here to avoid breaking users installations during upgrade.

Reply:
We can update the default on main because it was not released yet.